core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -2636,7 +2636,7 @@ public void testTableExistsViaHEADRequest() {
             any());
     Mockito.verify(adapter)
         .execute(
-            reqMatcher(HTTPMethod.HEAD, "v1/namespaces/newdb/tables/table", Map.of(), Map.of()),
+            reqMatcher(HTTPMethod.HEAD, RESOURCE_PATHS.table(TABLE), Map.of(), Map.of()),
```

Comment:
could you please update all the other places in this test class to using `RESOURCE_PATHS` instead of hardcoding the expected endpoint?

Reply:
For sure. I changed the all places that I can see